### PR TITLE
[ADD] website_forum_seo: add new module for improve forum que/ans description

### PR DESCRIPTION
**Pull request by imjaypatel**
<b>Task:</b> https://www.odoo.com/web#id=9107&view_type=form&model=project.task&menu_id=3942&action=327
<b>Pad:</b> https://pad.odoo.com/p/r.aAXbFLdZvu62soZi

<b>New Module:</b> website_forum_seo
<b>VIew Demo:</b> https://www.youtube.com/watch?v=_lYH_GrRqso

### CODE REVIEW

File: addons/website_forum_seo/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    'author': 'Odoo SA',
+    'depends': ['website_forum', 'marketing'],
+    'data': [
+        'views/forum_seo.xml',
+        'security/ir.model.access.csv',
+    ],
+    'demo': [
+        'data/forum_seo_demo.xml'
+    ],
+    'application': False,
```
Comment (atp-odoo) [inline addons/website_forum_seo/__openerp__.py]: Default value for 'application' is False. not required to declare here. and also comma is not required as last sentence. ;)

Comment (jpa-odoo) [inline addons/website_forum_seo/__openerp__.py]: ok, I remove this.

File: addons/website_forum_seo/tests/common.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from openerp.addons.website_forum.tests.common import TestForumCommon
+from openerp.tests import common
+
```
Comment (atp-odoo) [inline addons/website_forum_seo/tests/common.py]: unused import common

Comment (jpa-odoo) [inline addons/website_forum_seo/tests/common.py]: Yeah, I removed

**Pull request closed without merging**